_tests/e2e/config/browsers/remote_browsers.json

```diff
@@ -1,34 +1,42 @@
 {
+  "bs_mac_safari": {
+    "browserName": "Safari",
+    "browser_version": "11.0",
+    "os": "OS X",
+    "os_version": "High Sierra",
+    "resolution": "1024x768",
```

Comment:
If we end up using this for visual checks, we should probably include some other resolutions as well, but that's probably down the road a bit.

Reply:
Yeah don't think we need to worry about that at the moment. These are just browserstack capabilities. They can easily be reconfigured.